config/initializers/user_roles.rb

```diff
@@ -1,2 +1,3 @@
 Decidim.user_roles << 'department_admin'
+Decidim.user_roles << 'space_admin'
```

Comment:
@spuigmitjadxc what is this new role for?
This PR is supposed to show users with existing roles, not to create new roles.